### PR TITLE
feat(web.hosting) : new web hosting - Lot 2

### PR DESCRIPTION
**Pull request by karim-ovh**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-12083
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

New web Hosting - Lot 2 - WW ex US

### CODE REVIEW

**Review by darsene — DISMISSED** [review]

File: packages/manager/apps/web/client/app/hosting/database/order/public/components/steps/db-categories-offers/controller.js [hunk 1]
```diff
@@ -17,7 +17,14 @@ export default class WebHostingDatabaseOrderComponentsDbCategoriesOffersControll
   }
 
   getDbEngines() {
-    return this.model.dbCategory?.selectVersion?.engines || [];
+    return this.model.dbCategory?.selectVersion?.engines?.map((engine) => {
+      engine.versions.sort((a, b) => {
```
Comment (JacquesLarique) [inline packages/manager/apps/web/client/app/hosting/database/order/public/components/steps/db-categories-offers/controller.js:21]: I think you forgot a return here, also if engines is not set, getDbEngines will return undefined and not [] as previously

**Review by antonymarion — APPROVED** [review]

File: packages/manager/apps/web/client/app/hosting/database/order/public/components/steps/db-categories-offers/controller.js (hunk 1 above)
Comment (karim-ovh) [reply packages/manager/apps/web/client/app/hosting/database/order/public/components/steps/db-categories-offers/controller.js:21]: Oh yes i forgot something here ! OK i will add it.

**Review by JacquesLarique — DISMISSED** [review]

File: packages/manager/apps/web/client/app/hosting/database/order/public/components/steps/db-categories-offers/controller.js [hunk 2]
```diff
@@ -17,7 +17,15 @@ export default class WebHostingDatabaseOrderComponentsDbCategoriesOffersControll
...
-    return this.model.dbCategory?.selectVersion?.engines || [];
+    const engines = this.model.dbCategory?.selectVersion?.engines || [];
+    return engines.map((engine) => {
+      return engine.versions.sort((a, b) => {
+        // Order versions in descending order
+        return b.dbVersion.localeCompare(a.dbVersion, undefined, {
+          numeric: true,
+        });
+      });
+    });
```
Comment (anooparveti) [inline packages/manager/apps/web/client/app/hosting/database/order/public/components/steps/db-categories-offers/controller.js:28]: **nitpick:** Unnecessary local variable can be avoided
```suggestion
    return this.model.dbCategory?.selectVersion?.engines?.map((engine) => {
      return engine.versions.sort((a, b) => {
        // Order versions in descending order
        return b.dbVersion.localeCompare(a.dbVersion, undefined, {
          numeric: true,
        });
      });
    }) || [];
```

File: packages/manager/apps/web/client/app/hosting/general-informations/translations/Messages_de_DE.json
Comment (anooparveti) [inline packages/manager/apps/web/client/app/hosting/general-informations/translations/Messages_de_DE.json:1]: **nitpick:** remove the keys only from `Messages_fr_FR.json` file

File: packages/manager/apps/web/client/app/hosting/general-informations/translations/Messages_en_GB.json
Comment (anooparveti) [inline packages/manager/apps/web/client/app/hosting/general-informations/translations/Messages_en_GB.json:1]: **nitpick:** remove the keys only from `Messages_fr_FR.json` file

File: packages/manager/apps/web/client/app/hosting/general-informations/translations/Messages_es_ES.json
Comment (anooparveti) [inline packages/manager/apps/web/client/app/hosting/general-informations/translations/Messages_es_ES.json:1]: **nitpick:** remove the keys only from `Messages_fr_FR.json` file

File: packages/manager/apps/web/client/app/hosting/general-informations/translations/Messages_fr_CA.json
Comment (anooparveti) [inline packages/manager/apps/web/client/app/hosting/general-informations/translations/Messages_fr_CA.json:1]: **nitpick:** remove the keys only from `Messages_fr_FR.json` file

File: packages/manager/apps/web/client/app/hosting/general-informations/translations/Messages_it_IT.json
Comment (anooparveti) [inline packages/manager/apps/web/client/app/hosting/general-informations/translations/Messages_it_IT.json:1]: **nitpick:** remove the keys only from `Messages_fr_FR.json` file

File: packages/manager/apps/web/client/app/hosting/general-informations/translations/Messages_pl_PL.json
Comment (anooparveti) [inline packages/manager/apps/web/client/app/hosting/general-informations/translations/Messages_pl_PL.json:1]: **nitpick:** remove the keys only from `Messages_fr_FR.json` file

File: packages/manager/apps/web/client/app/hosting/general-informations/translations/Messages_pt_PT.json
Comment (anooparveti) [inline packages/manager/apps/web/client/app/hosting/general-informations/translations/Messages_pt_PT.json:1]: **nitpick:** remove the keys only from `Messages_fr_FR.json` file

**Review by anooparveti — CHANGES_REQUESTED** [review]

**Review by darsene — APPROVED** [review]

**Pull request merged**